cc_licenses/urls.py

```diff
@@ -29,6 +29,8 @@ def distill_no_parameters():
 urlpatterns = [
     url(r"^admin/", admin.site.urls),
     distill_path("", home, name="home", distill_func=distill_no_parameters),
+    url(r"status/(?P<id>\d+)/$", branch_status, name="branch_status"),
+    url(r"status/$", translation_status, name="translation_status"),
```

Comment:
I think we will have to include these pages in the build directory of various branches in cc-licenses-data as static files. Shouldn't they be using `distill_path`?

Reply:
Good catch! Will do.